packages/e2e/features/ui/components/storage/storage-browser/insufficient-permissions.feature

```diff
@@ -15,7 +15,7 @@ Feature: Storage Browser denied access without sufficient permissions
 
   @react
   Scenario: Attempt to add folder without write access
-    Given I intercept a "POST" request to "forbidden/Blackberry/?uploads"
+    Given I intercept a "PUT" request to "forbidden/Blackberry/"
```

Comment:
Need to listen to PUT request without `?uploads` since the uploadData call is made with data as ' ', which equates to 0 bytes and because of a recent fix we now correctly do a single part upload which is a PUT request without `?uploads`.

Link to the recent fix on the JS integrity branch that resulted in this change: https://github.com/aws-amplify/amplify-js/pull/13928/files#r1803424106